apc/proxy.php

```diff
@@ -9,12 +9,35 @@
 	}
 }
 
+/**
+ * Determines if SSL is used.
+ *
+ * @see is_ssl() (wp-includes/load.php).
+ *
+ * @return bool True if SSL, otherwise false.
+ */
+function cachify_is_ssl() {
+	if ( isset( $_SERVER['HTTPS'] ) ) {
+		if ( 'on' == strtolower( $_SERVER['HTTPS'] ) ) {
```

Comment:
I don't see a huge performance deficit or benefit here (the strtolower function also adds some runtime). But honestly, this is probably a time in nanoseconds and doesn't really sums up especially when called only once per request.

I would go for the strict comparison only because it's recommended/necessary due to the strict WordPress Coding Standards. And as the header will always be a string, the strict comparison should not break anything.